README.md

```diff
@@ -1,3 +1,65 @@
-# opencollective-changelog
+# Open Collective changelog
 
-To keep track of weekly changes in the code base
+We use this project to track weekly closed issues on [opencollective/opencollective](https://github.com/opencolllective/opencollective).
+
+## Live URL
+
+https://changelog.opencollective.com
+
+## Development
+
+### Prerequisites
+
+#### Node.js
+
+Make sure you have Node.js version >= 10. We recommend using [nvm](https://github.com/creationix/nvm): `nvm install`
+
+### Install
+
+```
+git clone https://github.com/opencollective/opencollective-changelog.git
+cd opencollective-changelog
+npm install
+```
+
+### Start
+
+`npm run dev`
+
+This will start your local copy of Changelog. You can access it at `http://localhost:3000/`
+
+## Updating Changelog
+
+To update the changelog in order and see the latest changes:
+
+- Run `npm run update:changelog`. The changelog will be updated.
+
+- The latest changes will be added to the log file, commit the file.
```

Comment:
Can you just give commands that anyone can copy paste?

Reply:
I don't get?